docs/GCBMDevelopmentSetup/windows_installation.rst

```diff
@@ -9,9 +9,11 @@ Before proceeding further, make sure you have the following prerequisites setup:
 
 Prerequisites
 -------------
-* `Python 3.7 <prerequisites/python_3.7.html>`_
-* `Microsoft Access Database Driver <prerequisites/microsoft_access_database_driver.html>`_
-* `Visual C++ Redistributable Packages <prerequisites/visual_c++_redistributable_packages.html>`_
+The prerequisites are:
+* Python 3.7
+* Microsoft Access Database Driver
+* Visual C++ Redistributable Packages
+You can view the details of the prerequisites `here. <https://docs.moja.global/en/master/GCBMDevelopmentSetup/prerequisites.html>`_
```

Comment:
@chicken-biryani 
Currently it looks like this:

> The prerequisites are: * Python 3.7 * Microsoft Access Database Driver * Visual C++ Redistributable Packages You can view the details of the prerequisites here.

Could you make it look like this?

>The prerequisites are: 
>
> - **Python 3.7**
> - **Microsoft Access Database Driver**  
> - **Visual C++ Redistributable Packages** 
> 
>Please set up the prerequisites by following the guide here.

Thanks! ✌

Reply:
Apologies, my bad!